AndelaOSP/art-android
# #156161391 Delete .idea from remote

#### What does this PR do?
Delete the .idea folder from remote
#### Description of Task to be completed?
- delete the files that have already been pushed to upstream which need to be ignored
- add those files to .gitignore
- ensure .idea doesn't on the remote
#### How should this be manually tested?
Check out to this branch and ensure all files are deleted and ignored successfully
#### What are the relevant pivotal tracker stories?
#156161391
